worker/Dockerfile

```diff
@@ -4,6 +4,10 @@ WORKDIR /browser_server
 
 RUN npx -y playwright@latest install --with-deps
 
+# TODO: figure out version issue
+RUN mv /root/.cache/ms-playwright/firefox-1490 /root/.cache/ms-playwright/firefox-1489 \
```

Comment:
Could you share the motivation behind it? I’m not sure why this is needed

Reply:
I think we need to bump the playwright library version because right now it looks for the paths I moved the directories to